l10n_ch_import_winbiz/wizard/l10n_ch_import_winbiz.py

```diff
@@ -0,0 +1,218 @@
+    state = fields.Char(sting="Import state"
+                        'Report',
+                        readonly=True,
+                        default="draft"
+                        )
+    file = fields.Binary(
+        'File',
+        required=True
+        )
+    imported_move_ids = fields.Many2many(
+        'account.move', 'import_cresus_move_rel',
+        string='Imported moves')
+
+    help_html = fields.Html('Import help', readonly=True,
+                            default=_('''
+                 In order to import your 'Winbiz Salaires' .xml \
+                 file you must complete the following requirements : </br>
+                * The accounts, analytical accounts used in the Cresus\
+                 file must be previously created into Odoo  </br>
+                * The date of the entry will determine the period used\
+                 in Odoo, so please ensure the period is created already. </br>
+                </br>'''))
+
+    HEAD_ODOO = ['ref', 'date', 'period_id', 'journal_id',
+                 'line_id/account_id', 'line_id/partner_id', 'line_id/name',
+                 'line_id/debit', 'line_id/credit',
+                 'line_id/account_tax_id',
+                 'line_id/analytic_account_id']
+
+    @api.multi
+    def open_account_moves(self):
+        res = {
+            'domain': str([('id', 'in', self.imported_move_ids.ids)]),
+            'name': 'Account Move',
+            'view_type': 'form',
+            'view_mode': 'tree,form',
+            'res_model': 'account.move',
+            'view_id': False,
+            'type': 'ir.actions.act_window',
+        }
+        return res
+
+    def format_messages(self, messages):
+        """Format error messages generated by the BaseModel.load method
+
+        :param messages: return of BaseModel.load messages key
+        :returns: formatted string
+
+        """
+        res = []
+        for msg in messages:
+            rows = msg.get('rows', {})
+            res.append(_("%s. -- Field: %s -- rows %s to %s") % (
+                msg.get('message', 'N/A'),
+                msg.get('field', 'N/A'),
+                rows.get('from', 'N/A'),
+                rows.get('to', 'N/A'))
+            )
+        return "\n \n".join(res)
+
+    def _manage_load_results(self, result):
+        """Manage the BaseModel.load function output and store exception.
+
+        Will generate success/failure report and store it into report field.
+        Manage commit and rollback even if load method uses PostgreSQL
+        Savepoints.
+
+        :param result: BaseModel.load returns
+                       {ids: list(int)|False, messages: [Message]}
+        """
+        # Import sucessful
+        if not result['messages']:
+            self.state = 'done'
+            self.report = _("Lines imported")
+            self.imported_move_ids = result['ids']
+        else:
+            self.report = self.format_messages(result['messages'])
+            self.state = 'error'
```

Comment:
Just a remark here, each attribute assignation will be a separate write. It is preferable to use write method when you have multiple fields to write. (wizard values are stored in database)